crates/libwebrtc-sys/src/lib.rs

```diff
@@ -557,6 +629,49 @@ impl RtpSenderInterface {
     }
 }
 
+/// This interface describes an ICE candidate, described in
+/// [RFC5245 Section 2][1].
+///
+/// [1]: https://datatracker.ietf.org/doc/html/rfc5245#section-2
+pub struct IceCandidateInterface(UniquePtr<webrtc::IceCandidateInterface>);
+
+impl IceCandidateInterface {
+    pub fn new(
+        sdp_mid: &str,
+        sdp_mline_index: i32,
+        candidate: &str,
+    ) -> anyhow::Result<IceCandidateInterface> {
+        let mut error = String::new();
+        let inner = webrtc::create_ice_candidate(
+            sdp_mid,
+            sdp_mline_index,
+            candidate,
+            &mut error,
+        );
+
+        if !error.is_empty() {
+            bail!(error);
+        }
+
+        Ok(IceCandidateInterface(inner))
+    }
+
+    #[must_use]
+    pub fn mid(&self) -> String {
+        webrtc::sdp_mid_of_ice_candidate(&self.0).to_string()
+    }
+
+    #[must_use]
+    pub fn candidate(&self) -> String {
+        webrtc::ice_candidate_interface_to_string(&self.0).to_string()
+    }
+
+    #[must_use]
+    pub fn mline_index(&self) -> i32 {
```

Comment:
Here and for the methods above the documantion is absent. Should be present.

Reply:
@tyranron Done.